### PR TITLE
chore: rename official pg to vanilla pg and upgrade to v1.0 api

### PR DESCRIPTION
**Pull request by kizuna-lek**

### CODE REVIEW

File: addons/vanilla-postgresql/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+maintainers:
+  - name: ApeCloud
+    url: https://github.com/apecloud/kubeblocks/
+
+sources:
+  - https://github.com/apecloud/kubeblocks/
+
+annotations:
+  addon.kubeblocks.io/kubeblocks-version: ">=0.9.0"
```
[the remark this answers is not in the document]
Comment (kizuna-lek) [reply addons/vanilla-postgresql/Chart.yaml]: done

**Review by shanshanying — APPROVED** [review]

File: addons/vanilla-postgresql/config/pg12-config-constraint.cue [hunk 2]
```diff
@@ -0,0 +1,921 @@
+//Copyright (C) 2022-2023 ApeCloud Co., Ltd
```
Comment (Y-Rookie) [inline addons/vanilla-postgresql/config/pg12-config-constraint.cue:1]: remove license

Comment (kizuna-lek) [reply addons/vanilla-postgresql/config/pg12-config-constraint.cue]: done

File: addons/vanilla-postgresql/templates/cmpv.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major14.minor07 }}
+    - name: 15.7.0
+      serviceVersion: 15.7.0
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major15.minor07 }}
+    - name: supabase-15.6.1.138
+      serviceVersion: 15.6.1138
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.supabase.repository }}:{{ .Values.image.supabase.tags.major15.minor61138 }}
```
Comment (Y-Rookie) [inline addons/vanilla-postgresql/templates/cmpv.yaml:41]: the image tag in cmpv should be static, check them all

File: addons/vanilla-postgresql/templates/cmpv.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    - name: 14.7.0
+      serviceVersion: 14.7.0
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major14.minor07 }}
+    - name: 15.7.0
+      serviceVersion: 15.7.0
+      images:
+        postgresql: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:{{ .Values.image.tags.major15.minor07 }}
+    - name: supabase-15.6.1.138
+      serviceVersion: 15.6.1138
```
Comment (Y-Rookie) [inline addons/vanilla-postgresql/templates/cmpv.yaml:39]: may be  15.6.1-138 would be better?

File: addons/vanilla-postgresql/templates/cmpv.yaml (hunk 3 above)
Comment (kizuna-lek) [reply addons/vanilla-postgresql/templates/cmpv.yaml]: done

File: addons/vanilla-postgresql/templates/cmpv.yaml (hunk 4 above)
Comment (kizuna-lek) [reply addons/vanilla-postgresql/templates/cmpv.yaml]: has been modified to 1-138

**Review by Y-Rookie — APPROVED** [review]

**Pull request merged**